open-metadata-implementation/access-services/subject-area/subject-area-api/pom.xml

```diff
@@ -30,8 +30,8 @@
     <dependencies>
 
         <dependency>
-            <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Just as a FYI I checked the dependency tree and it's not too bad as this is plain java code:
```
org.odpi.egeria:subject-area-api:jar:2.3-SNAPSHOT
 [INFO] +- org.odpi.egeria:open-metadata-types:jar:2.3-SNAPSHOT:compile
 [INFO] |  \- org.odpi.egeria:repository-services-archive-utilities:jar:2.3-SNAPSHOT:compile
 [INFO] |     +- org.odpi.egeria:open-metadata-archive-file-connector:jar:2.3-SNAPSHOT:compile
 [INFO] |     \- commons-io:commons-io:jar:2.7:compile
```
It's a 200k jar though it's dependenceis (esp commons-io) adds more - but probably typical for any of our clients.

Which really leaves any broader design discussions .. I'm less concerned with the inclusion per se?